spec/memoize_query_spec.rb

```diff
@@ -319,9 +319,9 @@ def expect_to_eq_with_or_without_redis
     end
   end
 
-  it 'memoizes queries with JOIN conditions' do
+  it 'does not memoize queries with JOIN conditions' do
     teacher = Teacher.create!
-    expect_to_use_redis do
+    expect_not_to_use_redis do
```

Comment:
Technically we can support this query though right? What's an example join query that wouldn't be supported well, as long as we have this [change](https://github.com/chanzuckerberg/redis-memo/pull/4) to make sure that a filtering condition exists on the table we're selecting?

Reply:
Great question! Currently, it's possible to select columns from a table that does not have any filtering conditions:
```SQL
SELECT "teacher_id" FROM "sites" INNER JOIN "teacher_sites" ON "sites"."id" = "teacher_sites"."site_id" WHERE "sites"."id" IN (...)
```